.github/workflows/linux.yml

```diff
@@ -25,31 +25,26 @@ jobs:
     name: Try to download prebuilt OpenVINO
     outputs:
       status: ${{ steps.openvino_download.outcome }}
+      ov_package_path: ${{ steps.openvino_download.outputs.ov_package_path }}
+      ov_wheel_input: ${{ steps.openvino_download.outputs.ov_wheel_input }}
+      ov_wheel_source: ${{ steps.openvino_download.outputs.ov_wheel_source }}
     timeout-minutes: 10
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04
+    runs-on: aks-linux-2-cores-8gb
+    container:
+      image: 'openvinogithubactions.azurecr.io/openvino_provider:0.1.0'
+      volumes: [ "/mount:/mount" ]
+    continue-on-error: true
 
     steps:
-      - name: Download OpenVINO build
-        id: openvino_download
-        run: |
-          wget ${{ env.OV_TARBALL}} --progress=bar:force:noscroll -O openvino_package.tar.gz
-          tar -tvf openvino_package.tar.gz
-        continue-on-error: true
-
-      #
-      # Upload to artifacts
-      #
-
-      - name: Upload openvino package
-        if: steps.openvino_download.outcome == 'success'
-        uses: actions/upload-artifact@v4
-        with:
-          name: openvino_package
-          path: openvino_package.tar.gz
-          if-no-files-found: 'error'
+    - uses: openvinotoolkit/openvino/.github/actions/openvino_provider@master
+      id: openvino_download
+      with:
+        ov_artifact_name: ${{ env.OV_ARTIFACT_NAME }}
+        commit_build_key: 'public_linux_ubuntu_20_04_x86_64_release'
```

Comment:
will we be able to dynamically switch debug type?

Reply:
for debug mode, we still can use release builds on Linux.

Reply:
You mean, like that?
`public_linux_ubuntu_20_04_x86_64_{env.BUILD_TYPE}`? yes

Reply:
discussed offline to simplify this key (os: ubuntu, arch: x86_64, etc)